dev/com.ibm.ws.jsp.2.3_fat/test-applications/PH62212.war/resources/large-jsp.jsp

```diff
@@ -0,0 +1,2217 @@
+<!—
```

Comment:
Start and end tags here look incorrect.

Reply:
Also how was this JSP generated?

Reply:
As mentioned in our DMs, I reused https://github.com/OpenLiberty/open-liberty/blob/06d4e618f54adbd7b225eae17634a64f8826f837/dev/com.ibm.ws.jsp.2.3_fat/test-applications/TestJSTL.war/resources/coreTags.jsp and copied/pasted it repeatedly